portfolio/src/main/webapp/js/main.js

```diff
@@ -23,16 +23,55 @@ $(document).ready(function(){
 })
 
 function getComments() {
-  let max = document.getElementById('num-comments').value;
-  fetch('/list-comments?max=' + max).then(response => response.json()).then((comments) => {
-    const commentsListElement = document.getElementById('comments-container');
-    commentsListElement.innerHTML = '';
-    comments.forEach(comment => {
-      commentsListElement.appendChild(createCommentElement(comment));
+  let numCommentsPerPage = document.getElementById('num-comments').value;
+  let numPages = setupPagination(numCommentsPerPage);
```

Comment:
We could inline `numPages` here since it is not used anywhere else.

Reply:
changed